src/storage/src/hummock/local_version_manager.rs

```diff
@@ -511,35 +508,92 @@ impl LocalVersionManager {
         for result in join_all(join_handles).await {
             result.expect("should be able to join the flush handle")
         }
-        let (order_index, task_payload, task_write_batch_size) = match self
-            .local_version
-            .write()
-            .get_mut_shared_buffer(epoch)
-            .and_then(|shared_buffer| shared_buffer.new_upload_task(SyncEpoch))
-        {
-            Some(task) => task,
-            None => {
-                tracing::trace!("sync epoch {} has no more task to do", epoch);
-                return Ok(0);
+        let sync_epoch_notify_clone = self.sync_epoch_notify.clone();
```

Comment:
clone is not needed. You can just do self.sync_epoch_notify.notified().await in L524.